mappings/net/minecraft/util/math/GlobalPos.mapping

```diff
@@ -0,0 +1,14 @@
+CLASS fb net/minecraft/util/math/GlobalPos
```

Comment:
`UniversalPos`? `Global` does not fit the minecraft theme.

Reply:
The Mojang name is `GlobalPos`, it seems.